setup.md

```diff
@@ -221,6 +221,20 @@ To develop Flutter apps for Android, you can use either a Mac or a Linux (64-bit
 
 1. Start Android Studio.
 
+### Set up the Android emulator
+
+To prepare to run and test your Flutter app on the Android emulator, follow these steps:
+
+1. In **Android Studio>Configure>SDK Manager**, install the system image(s) for the Android
+version you want to emulate.
+1. Launch **Android Studio>Tools>Android>AVD Manager** to define an Android Virtual Device (AVD).
```

Comment:
Or `x86_64`, and until @chinmaygarde turns on the software backend by default, they need to enable hardware acceleration by checking "Use Host GPU".